charms/kubernetes_snaps.py

```diff
@@ -559,32 +570,68 @@ def get_sandbox_image(registry) -> str:
     return f"{registry}/pause:3.9"
 
 
+def get_snap_version(name: str) -> str:
+    """
+    Get the version of an installed snap package.
+
+    Args:
+    name (str): The name of the snap package.
+
+    Returns:
+    str or None: The version of the snap package if available, otherwise None.
+    """
+    cmd = ["snap", "list", name]
+    result = check_output(cmd)
+    output = result.decode().strip()
+    match = re.search(r"\b\d+(?:\.\d+)*\b", output)
+
+    if match:
+        version = match.group()
+        return version
```

Comment:
you may want to not name this `version` these so they don't confuse someone thinking it's the imported `version` from `packaging`

```suggestion
    if match:
        return match.group()
```